src/main/java/net/openhft/chronicle/jlbh/ImmutableJLBHResult.java

```diff
@@ -25,7 +25,7 @@
 
 import static java.util.Collections.unmodifiableMap;
 
-class ImmutableJLBHResult implements JLBHResult {
+final class ImmutableJLBHResult implements JLBHResult {
```

Comment:
Classes are packet private so we know they are never subclassed externally

Reply:
ugg !

Reply:
you know what I think of final classes! :-)

Reply:
I agree with you Rob!

Reply:
See the subject. If a class is extendable, then, by definition, instances of the class can exist, that is mutable.